### ISSUE TITLE
Error Installing on Mac OSX - Sierra - ZSH, OhMyZsh, iTerm

### ISSUE THREAD

**Issue by imjohsep** [howtowhale/dvm#165]
If you are reporting a bug, please use the template below:

### What didn't work?
Installation failed with both brew and `curl -sL https://howtowhale.github.io/dvm/downloads/latest/install.sh | sh`

```
$ dvm 
zsh: command not found: dvm
# added `[ -f /usr/local/opt/dvm/dvm.sh ] && . /usr/local/opt/dvm/dvm.sh` to profile
$ source ~/.zshrc
/usr/bin/cd: line 4: cd: -q: invalid option
cd: usage: cd [-L|-P] [dir]
```

After changing the -q flag in dvm.sh to a -P the following occured
```
$ dvm
Installation corrupt: dvm-helper is missing. Please reinstall dvm.
```

The result of DVM_HELPER in dvm.sh is: `/Users/<user_name>/dvm-helper/dvm-helper`
Obviously, user_name is my user name.

I then hard coded the DVM_HELPER to be `/Users/<user_name>/.dvm/dvm-helper/dvm-helper`

### What should it have done?

It should have installed correctly with the proper DVM_HELPER path.

### Helpful Context
If you can provide the following info, it will help us figure things out more quickly! :smile:

What version of dvm are you using?

```
$ dvm --version
Docker Version Manager version 0.8.3 (62dc2cb)
```

What is your Operating System?

Mac OSX Sierra

What is your shell (bash/powershell/cmd/fish)?

ZSH

How did you install dvm (command-line,chocolatey,homebrew)?

command-line and homebrew

Do you like kittens? :smile_cat:

Of course! [Bubs](https://i.ytimg.com/vi/BDL1nFROrEM/maxresdefault.jpg)

**Comment by carolynvs** [comment]
Thanks for the bug report. If anyone with time and ZSH would like to work on a fix, the help would be welcome! 

Related:
https://github.com/howtowhale/dvm/pull/130#issuecomment-289874612
https://github.com/howtowhale/dvm/issues/144

**Comment by carolynvs** [comment]
I think this has been fixed by #176.
